packages/react-native-client/ios/RNFishjamClient.swift

```diff
@@ -327,6 +344,10 @@ class RNFishjamClient: FishjamClientListener {
     func toggleScreenShare(screenShareOptions: ScreenShareOptions) throws {
         try ensureCreated()
         try ensureConnected()
+        guard isAppScreenShareOn == false else {
```

Comment:
**suggestion:** 
```suggestion
        guard !isAppScreenShareOn else {
```

Reply:
In my opinion it is more readable to use in this case comparison to false than negating the value.